install_zeromq.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+FILE=/home/travis/lib/libzmq.so
```

Comment:
does this work on drone?

Reply:
yes, this script will be used in drone,too.

Reply:
but this assumes there's a "travis" user?

Reply:
yes, also drone.io uses it